### ISSUE TITLE
Different behaviors depending on the browser

### ISSUE THREAD

**Issue by cyril-sf** [tildeio/oasis.js#15]
Using MessageChannel.js, it is possible to send more complex objects on IE/Firefox than with Chrome.

**Comment by hjdivad** [comment]
@Cyril-sf example?

Is it that Kamino supports self-references but stock `postMessage` does not?

**Comment by cyril-sf** [comment]
Yes, it comes from the fact that MessageChannel.js relies on Kamino.js.

For a comprehensive list, you can refer to [Kamino.js page](https://github.com/tildeio/kamino.js#differences-from-json)
